draft-ietf-quic-transport.md

```diff
@@ -1065,6 +1066,13 @@ sending a packet with a number of 0x6b2d79 requires a packet number encoding
 with 14 bits or more; whereas the 30-bit packet number encoding is needed to
 send a packet with a number of 0x6bc107.
 
+An endpoint MUST discard packets with duplicate packet numbers.  After
```

Comment:
How about combining the first two sentences, since they seem largely redundant? Maybe "An endpoint MUST discard packets with duplicate packet numbers after unprotecting the packet." or something similar.